src/OrchardCore.Modules/OrchardCore.Layers/ViewModels/LayersOptionsViewModel.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace OrchardCore.Layers.ViewModels
+{
+    public class LayersOptionsViewModel
```

Comment:
I'm sorry I removed my comment and you did not see it. I'm not sure this is the appropriate way to do this.